### DISCUSSION TITLE
filament-curator installation / filament-curator styles

### DISCUSSION THREAD

**Discussion by morisfa2** [awcodes/filament-curator#496] [Q&A]
Hello , i am installing the package in filamentphp v3 and is not showing the Media Manu in navigations list 

Why ? 
Cab anyone help to how install ? 

The read me not working for me

**Comment by awcodes** [comment]
Make sure to add the `CuratorPlugin::make()` to your panel provider

**Comment by morisfa2** [comment]
thanks worked
@awcodes

when i add 
->plugins([
                \Awcodes\Curator\CuratorPlugin::make()
            ])
is working

but with:

 ->plugins([
            \Awcodes\Curator\CuratorPlugin::make()
                ->label('Media')
                ->pluralLabel('Media')
                ->navigationIcon('heroicon-o-photo')
                ->navigationGroup('Content')
                ->navigationSort(3)
                ->navigationCountBadge()
                ->registerNavigation(false)
                ->defaultListView('grid' || 'list')
                ->resource(\App\Filament\Resources\CustomMediaResource::class)
        ]);

is not worked

but finally is done , thank you for creating this plugin as free

**Comment by awcodes** [comment]
Not sure what exactly isn't working, but this part is wrong it should be either 'grid' or 'list'.
```php
->defaultListView('grid' || 'list')
```

Don't just copy / paste from the docs, that is just meant to show what is possible. :)

**Comment by morisfa2** [comment]
![image](https://github.com/awcodes/filament-curator/assets/136106565/8448590e-4f92-458c-bbe3-81cc75e5ec3e)
![Screenshot from 2024-06-15 02-10-25](https://github.com/awcodes/filament-curator/assets/136106565/c14b14cd-0464-4bab-8615-4677d4a96644)

yes problem was exactly from ->defaultListView('grid' || 'list') , I didn't have enough attention to detail.


Thank you very much for your fast support on filamentphp



I have another problem right now, my added media styles are broken, can you help how to fix it?

this is my app.css

@import "../../vendor/awcodes/filament-curator/resources/css/plugin.css";

@tailwind base;
@tailwind components;
@tailwind utilities;

npm run build

**Reply by awcodes** [reply]
You also need to include the views in your themes tailwind config
